local-cluster/src/CardanoClient.hs

```diff
@@ -416,3 +418,17 @@ generatePaymentToCommit networkId (RunningNode _ nodeSocket) sk vk lovelace = do
   amountLovelace = Lovelace $ fromIntegral lovelace
 
   convertUtxo (UTxO ledgerUtxo) = Utxo ledgerUtxo
+
+mkSeedPayment :: NetworkId -> ProtocolParameters -> Lovelace -> RunningNode -> Cardano.SigningKey -> Lovelace -> IO Utxo
+mkSeedPayment networkId pparams initialAmount (RunningNode _ nodeSocket) sk amountLovelace = do
```

Comment:
Could: Rename this function. We seem to be using `mk..` function names to build things (not too happy about that convention though and it's nowhere written), while this function does create a transaction, sign, submit and even wait for it's effect. Maybe `makeSeedPayment`? :sweat_smile:

Reply:
Also, we always `void` this function (right now). We might want to consider returning `IO ()`

Reply:
This is probably because of me :grimacing: ... I tend to use `mk` a lot for **pure** smart constructors, and **new** for computation in `IO` or with side-effects that allocate resources.

Reply:
I just needed a function to make this seed payment, and there was one already but which was not doing the "right thing" and I stole the name but changed the behaviour.